core/src/main/java/com/scalar/db/util/ScalarDbUtils.java

```diff
@@ -249,4 +250,23 @@ public static boolean isRelational(Scan scan) {
     return scan instanceof ScanAll
         && (!scan.getOrderings().isEmpty() || !scan.getConjunctions().isEmpty());
   }
+
+  public static JDBCType getJdbcType(int sqlTypes) {
+    JDBCType type;
+    switch (sqlTypes) {
+      case 100:
```

Comment:
100 and 101 are BINARY_FLOAT and BINARY_DOUBLE, respectively? How about adding comments about that?

Reply:
Fixed in 0d3a2f8.